web/modules/custom/contact_information/contact_information.info.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
 config_devel:
   - core.entity_form_display.rdf_entity.contact_information.default
   - core.entity_form_display.rdf_entity.contact_information.inline
+  - core.entity_view_display.rdf_entity.contact_information.about
```

Comment:
> In wireframe, grey areas are 1 area (owner and contact)
I think that it is better if the fields are separated, it would be not consistent if some fields would have margin and some don't have.

> when multiple items in lists, markup is not showing a list, it's not the best semantically
It is default template from the drupal core, and there are divs instead of list.